package.json

```diff
@@ -52,6 +52,7 @@
     "grunt-plato": "1.2.1",
     "grunt-processhtml": "0.3.8",
     "grunt-shell": "1.1.2",
+    "grunt-svgmin": "^3.1.1",
```

Comment:
We pinned all other dependencies. So you should get rid of `^` for `grunt-svgmin`.